contracts/NonfungiblePositionManager.sol

```diff
@@ -96,7 +96,7 @@ contract NonfungiblePositionManager is INonfungiblePositionManager, BaseLiquidit
         // mint receipt token
         uint256 tokenId;
         _mint(owner, (tokenId = nextTokenId++));
-        tokenPositions[tokenId] = TokenPosition({owner: owner, range: range});
+        tokenPositions[tokenId] = TokenPosition({owner: owner, range: range, operator: address(0x0)});
```

Comment:
if you mint, maybe you should be allowed to specify a default operator? maybe thats wack though lol